src/applications/representative-search/components/results/ReportModal.jsx

```diff
@@ -82,12 +81,17 @@ const ReportModal = ({
     // push non-null items to reports object
     Object.keys(reportObject).forEach(prop => {
       if (reportObject[prop] !== null) {
-        formattedReportObject.reports[prop] = reportObject[prop];
+        if (prop === 'phone') {
+          formattedReportObject.reports[snakeCase('phoneNumber')] =
```

Comment:
is `snakeCase()` doing anything else here besides converting `phoneNumber` to `phone_number`? Since `phoneNumber` is already a hard coded string, can it just be switched to `phone_number` and the lodash import removed?